library/acl.inc

```diff
@@ -828,7 +828,9 @@
             return false;
         elseif($count_user_allowed > 0)
             return true;
-        elseif($count_group_denied > 0)
+        // We do not exclude user that belongs to group A and B
+        // In case of A group have permissions and B group does not have permissions
+        elseif($count_group_denied > 0 && $count_group_allowed < 1)
```

Comment:
sorry Eyal has committed this to fix a bug as explained in the comment :  

We want  to  include user that belongs to group A and group B
and return true in case of  group A have permissions and group B does not have permissions

fixed the comment in the branch